tests/sync/session.cpp

```diff
@@ -64,6 +79,20 @@ std::shared_ptr<SyncSession> sync_session(SyncServer& server, std::shared_ptr<Sy
 
 namespace {
 
+void add_objects(Realm::Config& config, int count=2)
```

Comment:
Since we only use this in one place, why not just do it in-line? Do we think other tests will need these same test objects?